modules/bind-server/main.tf

```diff
@@ -97,6 +97,8 @@ resource "aws_instance" "bind" {
 
   provisioner "remote-exec" {
     connection {
+      type                = "ssh"
+      agent               = true
```

Comment:
It looks like this is disabled by default - https://www.terraform.io/docs/provisioners/connection.html#agent - will enabling `agent` lead to failure if `ssh-agent` is not available/setup in the host/environment running Terraform?

Reply:
If you don’t have an agent, you wouldn’t be able to forward the key for the next hop with the bastion anyway, isn’t that correct?

Reply:
Is a bastion required? What about when you aren't using one?

Reply:
You are right! Removed `agent = true`.